src/decorator.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
```

Comment:
- [ ] I would recommend using `require_relative` instead of `require` for importing files or methods from the same folder structure. Please fix this here and all the files.